Clinical-Genomics/scout
# Fix #1343. Allow collaborators to view gene-variants.

This PR adds a functionality / removes a limitation.

**How to test**:
1. On a populated server, search for a gene in the gene-variants view and note that cases that belong to collaborator cases are not shown (e.g. as Stella G). If in doubt about cases shared, as another user, share a case with a known variant with an institute that Stella belongs to. <img width="407" alt="Screenshot 2019-11-01 at 13 33 49" src="https://user-images.githubusercontent.com/758570/68026534-4eeb5500-fcb0-11e9-8ff7-b77a44402bd6.png">
2. After updating to this branch, assure that variants from the previously shared case can now be seen on "All variants" by Stella G. <img width="759" alt="Screenshot 2019-11-01 at 14 02 27" src="https://user-images.githubusercontent.com/758570/68026535-4f83eb80-fcb0-11e9-9911-39e79149cac2.png">

**Expected outcome**:
The functionality should be working

**Review:**
